test/cmd/unix/test_cmd_sudo.py

```diff
@@ -62,6 +63,37 @@ def test_failing_with_timeout(buffer_connection, command_output_and_expected_res
         cmd_sudo(timeout=0.1)
 
 
+def test_can_use_timeout_of_embedded_command(buffer_connection, command_output_and_expected_result_timeout):
+    command_output = command_output_and_expected_result_timeout
+    buffer_connection.remote_inject_response([command_output])
+    cmd_pwd = Pwd(connection=buffer_connection.moler_connection)
+    cmd_pwd.timeout = 0.2
+    cmd_sudo = Sudo(connection=buffer_connection.moler_connection, password="pass", cmd_object=cmd_pwd)
+    cmd_sudo.terminating_timeout = 0  # no additional timeout for Ctrl-C..till..prompt (shutdown after cmd timeout)
+    start_time = time.time()
+    with pytest.raises(CommandTimeout):
+        cmd_sudo()
+    duration = time.time() - start_time
+    assert duration > 0.2
```

Comment:
`duration >= 0.2` ? It means no less than 0.2s ?

Reply:
you are right - should be `duration >= 0.2` - you can fix it